tileserver/mapnik_config/popluarity-score.conf

```diff
@@ -0,0 +1,4 @@
+name=popularity-score
+tiledir=/var/lib/tirex/tiles/popularity-score
+maxz=18
+mapfile=/home/sfuehr/Documents/TUB-WI/S7_BA_SimRa/simra-visualizations-server/tileserver/mapnik_maps/popularity-score.xml
```

Comment:
keine absoluten pfade benutzen, relative pfade oder sinnvolle absolute pfade die auf jedem System funktionieren

Reply:
Ich hatte im docker Setup die Maps einfach in /etc/tirex/maps/mapnik gepackt. Denke das ist die schönste Lösung wenn man die configs eh in /etc/tirex/renderer/mapnik packt. Symlinken kann man sich das ja irgendwo anders hin…